python/pyspark/sql/tests/test_pandas_udf_scalar.py

```diff
@@ -14,7 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import datetime
```

Comment:
Hi, @Fokko . The PR title is too broad. Is this fixing all `duplicate` imports?

Reply:
Yes, this is fixing all the duplicate imports:
```
fokkodriesprong@Fan spark % mypy python/ | grep -i "already defined (by an import)"
python/pyspark/sql/tests/test_pandas_udf_scalar.py:25: error: Name 'datetime' already defined (by an import)
```

Reply:
Great! Then, it's a perfect title.